contrib/builder/rpm/ppc64le/build.sh

```diff
@@ -3,6 +3,8 @@ set -e
 
 cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
+wget http://download.opensuse.org/repositories/Virtualization:/containers:/images:/openSUSE-Tumbleweed/images/openSUSE-Tumbleweed-docker-guest-docker.ppc64le.tar.xz && unxz openSUSE-Tumbleweed-docker-guest-docker.ppc64le.tar.xz && docker load -i openSUSE-Tumbleweed-docker-guest-docker.ppc64le.tar 
```

Comment:
Yah, let's get rid of that line. I was testing with tumbleweed before I switched to the leap releases.